docs/reference-guides/providers/capa.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 0
+---
+
+# Create & import a cluster using CAPI provider for AWS
+
+Remember that most Cluster API Providers are upstream projects maintained by the Kubernetes open-source community.
+
+## Prerequisites
+
+- Rancher Manager cluster with Rancher Turtles installed
+- Cluster API providers installed for your scenario - we'll be using the AWS infrastructure in these instructions - you can find a guide on how to install a provider using the `CAPIProvider` resource [here](../../tasks/capi-operator/basic_cluster_api_provider_installation.md)
+- **clusterctl** CLI - see the [releases](https://github.com/kubernetes-sigs/cluster-api/releases)
+
+## Create Your Cluster Definition
+
+To generate the YAML for the cluster, do the following (assuming the Docker infrastructure provider is being used):
+
+1. Open a terminal and run the following:
+
+```bash
+export KUBERNETES_VERSION=v1.28
+export AWS_REGION=eu-west-2
+export AWS_INSTANCE_TYPE=t3.medium
+
+clusterctl generate cluster cluster1 \
+--from https://raw.githubusercontent.com/rancher-sandbox/rancher-turtles-fleet-example/templates/capa.yaml \
```

Comment:
just a note, we're going to replace these examples with clusterclasses later

Reply:
There's a cluster class specific branch in the examples repo, which is probably outdated, but I'll take a look at it so we can use it in later iterations of the docs.